docs/content/guides/migrations/migrating-airflow-to-dagster.mdx

```diff
@@ -5,223 +5,242 @@ description: "Learn how do a lift-and-shift migration of airflow to Dagster."
-)
+from airflow import DAG
+from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+from pendulum import datetime
+
+with DAG(dag_id="example_kubernetes_dag", schedule_interval=None, catchup=False) as dag:
+    KubernetesPodOperator(
+        image="example-data-pipeline:latest",
+        name="example-kubernetes-task",
+        task_id="example-kubernetes-task",
+        get_logs=True,
+    )
 ```
 
----
-
-## Step 7: Move to production
+Now, let’s show how the same image would be run on Kubernetes using Dagster pipes and the `dagster_k8s` wrapper.
 
-<Note>
-  This step is applicable to Dagster+. If deploying to your infrastructure,
-  refer to the <a href="/deployment">Open Source Deployment guides</a> for more
-  info.
-  <br />
-  <br />
-  Additionally, until your Airflow DAGs execute successfully in your local environment,
-  we recommend waiting to move to production.
-</Note>
-
-In this step, you'll set up your project for use with Dagster+.
-
-<!-- Once you have your dagster cloud organization setup you can either choose to adapt your existing airflow repository to use the Dagster+ CI/CD deployment or copy over your code to the quickstart repository you made as part of the Dagster+ onboarding. -->
-
-1. Complete the steps in the [Dagster+ Getting Started guide](/dagster-plus/getting-started), if you haven't already. Proceed to the next step when your account is set up and you have the `dagster-cloud` CLI installed.
-
-2. In the root of your project, create or modify the [`dagster_cloud.yaml` file](/dagster-plus/managing-deployments/dagster-cloud-yaml) with the following code:
-
-   ```yaml
-   locations:
-     - location_name: dagster_migration
-       code_source:
-         python_file: dagster_migration.py
-   ```
-
-3. Push your code and let the CI/CD for Dagster+ run out a deployment of your migrated DAGs to cloud.
+```python
+from dagster import AssetExecutionContext, asset
+from dagster_k8s import PipesK8sClient
+
+
+@asset
+def k8s_pipes_asset(context: AssetExecutionContext, k8s_pipes_client: PipesK8sClient):
+    return k8s_pipes_client.run(
+        context=context,
+        image="example-data-pipeline:latest",
+        base_pod_spec={
+            "containers": [
+                {
+                    "name": "data-processing-rs",
+                    "image": "data-processing-rs",
+                }
+            ]
+        },
+    ).get_materialize_result()
+```
 
----
+The primary difference between Airflow and Dagster are how the k8s pod specification are expose. In Airflow, they are passed as parameters the `KubernetesPodOperator`, whereas in Dagster they are passed as a `base_pod_spec` dictionary to the `k8s_pipes_client.run` method. Another difference is that in Airflow, `get_logs` is required to capture _stdout_, however, with Dagster they are automatically captured to on the _stdout_ tab of the step output.
 
-## Step 8: Migrate permissions to Dagster
+**Advanced pipes usage**
```

Comment:
This should be a subheading, but I'm not sure what level is should be at the moment. We can fix it once we figure out what the structure of this page is going to look like.